examples/1-eks-cluster-with-new-vpc/main.tf

```diff
@@ -33,6 +35,128 @@ data "aws_eks_cluster_auth" "cluster" {
   name = module.aws-eks-accelerator-for-terraform.eks_cluster_id
 }
 
+data "aws_iam_session_context" "current" {
+  arn = data.aws_caller_identity.current.arn
+}
+
+data "aws_iam_policy_document" "eks_key" {
+
```

Comment:
I thought you are moving this policy to core module `data.tf` file. This might be too much in the example. We can create opinionated policy with best practices